docs/_docs/0_7_cos_filesystem.cn.md

```diff
@@ -0,0 +1,73 @@
+---
+title: COS Filesystem
+keywords: hudi, hive, tencent, cos, spark, presto
+permalink: /cn/docs/cos_hoodie.html
+summary: In this page, we go over how to configure Hudi with COS filesystem.
+last_modified_at: 2020-04-21T12:50:50-10:00
+language: cn
+---
+这个页面描述了如何让你的Hudi spark任务使用Tencent Cloud COS存储。
+
+## Tencent Cloud COS 部署
+
+为了让Hudi使用COS，需要增加两部分的配置:
+
+- 为Hidi增加Tencent Cloud COS的相关配置
+- 增加Jar包的MVN依赖
+
+### Tencent Cloud COS 相关的配置
+
+新增下面的配置到你的Hudi能访问的core-site.xml文件。使用你的COS bucket name替换掉`fs.defaultFS`，使用COS的key和secret分别替换`fs.cosn.userinfo.secretKey`和`fs.cosn.userinfo.secretId`。主要Hudi就能读写相应的bucket。
+
+```xml
+    <property>
+        <name>fs.defaultFS</name>
+        <value>cosn://examplebucket-1250000000</value>
+    </property>
+
+    <property>
+        <name>fs.cosn.userinfo.secretId</name>
+        <value>xxxxxxxxxxxxxxxxxxxxxxxxx</value>
```

Comment:
cos-secret

Reply:
Thank you for the suggestion.  I have optimized the desc.